### DISCUSSION TITLE
Renaming Dockerfile

### DISCUSSION THREAD

**Discussion by preone** [Checkmarx/kics#6339] [Q&A] [answered]
Renaming Dockerfile with a dot can bypass the scan.

Rename Dockerfile to whatever it parses as a dockerfile.
Rename Dockerfile to whatever.txt it ignores it.
Rename Dockerfile to whatever.txt.dockerfile it parses as a dockerfile.

In a folder with several dockerfiles with different extensions kics is going to ignore them. Is there a way to force the scan with files with dots (all extensions)?

**Comment by gabriel-cx** [comment, marked as answer]
Hi @preone ,

Thank you for your inputs.
Kindly check [here](https://docs.kics.io/latest/platforms/#docker) which files KICS recognize as Docker files.
